app.py

```diff
@@ -240,87 +252,77 @@ def generate_table(state, day_type, period, orientation='ew'):
-                            font={'color':BASELINE_LINE['color']},
-                            xref='paper',
-                            yref='yaxis',
-                            showarrow=False
-                            )]
-        line = {'type':'line',
-                'x0': 0,
-                'x1': 1,
-                'xref': 'paper',
-                'y0': base_data.iloc[0]['tt'],
-                'y1': base_data.iloc[0]['tt'],
-                'line': BASELINE_LINE
-               }
-        layout = dict(font={'family': FONT_FAMILY},
-                      autosize=True,
-                      height=225,
-                      barmode='relative',
-                      xaxis=dict(title='Date',
-                                 range=DATERANGE,
-                                 fixedrange=True),
-                      yaxis=dict(title='Travel Time (min)',
-                                 range=[0, MAX_TIME[orientation]],
-                                 fixedrange=True),
-                      shapes=[line],
-                      margin=PLOT['margin'],
-                      annotations=annotations,
-                      legend={'xanchor':'right'}
-                      )
-        figdiv = html.Div(dcc.Graph(id = graph_name,
-                                    figure = ({'layout': layout, 'data': data}),
-                                    config={'displayModeBar': False}))
-    return figdiv
+        return None
+    data = [generate_graph_data(after_data,
+                                marker=dict(color=PLOT_COLORS['pilot']),
+                                name = 'Pilot',
+                                text=after_data['tt'].round(),
+                                hoverinfo='x+y',
+                                textposition='inside',
+                                insidetextfont=dict(color='rgba(255,255,255,1)',
+                                                    size=12)),
+            generate_graph_data(ghost_data,
+                                marker = dict(color = PLOT_COLORS['ghost_bar'],
+                                              line = dict(width = 2, 
+                                                          color = PLOT_COLORS['ghost_line'])),
+                                hoverinfo = 'skip',
+                                name = 'Ghost')]
+
+    annotations = [dict(x=-0.008,
+                        y=base_data.iloc[0]['tt'] + 2,
+                        text = 'Baseline',
+                        font={'color':BASELINE_LINE['color']},
+                        xref='paper',
+                        yref='yaxis',
+                        showarrow=False
+                        )]
+    base_line = {'type':'line', #baseline line
+            'x0': 0,
+            'x1': 1,
+            'xref': 'paper',
+            'y0': base_data.iloc[0]['tt'],
+            'y1': base_data.iloc[0]['tt'],
+            'line': BASELINE_LINE
+           }
+    axis_line = {'type':'line', #axis line
```

Comment:
Could you add a comment about why this is here (and specify which axis). Is it because the white bars cover the x-axis?